R/data_color.R

```diff
@@ -129,7 +129,7 @@ data_color <- function(data,
                        columns,
                        colors,
                        alpha = NULL,
-                       apply_to = "bkgd",
+                       apply_to = "fill",
```

Comment:
Note: BREAKING change. At the very least, consider putting a prominent note in NEWS. Consider checking for `"bkgd"` in the body of the function, and respond appropriately (maybe warn, then act as if they asked for `"fill"`?).

Reply:
Same with the move from `cells_styles` to `cell_text` and `cell_fill`, do you have a plan for how to notify?